code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -31,6 +31,20 @@
 	icon_state = "psychbed"
 	buildstackamount = 5
 
+/obj/structure/bed/wooden_lounge_chair
+	name = "wooden lounge chair"
+	desc = "Time to lay back and soak in the solar rays."
+	icon_state = "wooden_lounge_chair"
+	can_buckle = TRUE
+	anchored = TRUE
+	buckle_lying = TRUE
+	resistance_flags = FLAMMABLE
+	max_integrity = 100
+	integrity_failure = 30
```

Comment:
All of these variables are the same on the parent `/obj/structure/bed`, so you can remove them.